odh-manifests/osc-cl1/trino/base/trino-config-secret.yaml

```diff
@@ -92,23 +92,23 @@ stringData:
     group-provider.name=file
     file.group-file=/etc/trino/group-mapping.properties
   group-mapping.properties: |-
-    admins:admin,erik,hukhan,mikhail,ricardo,vincent,erikerlandson,caldeirav,HumairAK
+    admins:admin,erikerlandson,caldeirav,HumairAK,redmikhail
     aicoe_osc_demo:oindrillac,aakankshaduggal,shreyanand,michaelclifford,chauhankaranraj,sankbad,isabelizimm,harshad16,durandom,erikerlandson
     os_climate_corporate_data_project:ludans,caldeirav,DavWinkel,rmorais1,andraNew,ChristianMeyndt,ttschmucker,toki8,LeaADeleris,JeremyGohBNP,amirqamar
     climate_itr_tool_project:joriscram,alexanu,caldeirav,DavWinkel,ChristianMeyndt,hbaltzell,ttschmucker,toki8,LeylaJavadova,christian-schellnegger,roland-mf,franz-mf,EWinter21
-    data_commons_project:erik,caldeirav,MichaelTiemannOSC,hukhan,erikerlandson
+    data_commons_project:caldeirav,MichaelTiemannOSC,erikerlandson
   access-control.properties: |-
     access-control.name=file
     security.config-file=/etc/trino/rules.json
   rules.json: |-
     {
       "catalogs": [
       {
-        "group": "admins|aicoe_osc_demo|data_commons_project",
+        "group": "admins|data_commons_project",
         "allow": "all"
       },
       {
-        "group": "os_climate_corporate_data_project|climate_itr_tool_project",
+        "group": "aicoe_osc_demo|os_climate_corporate_data_project|climate_itr_tool_project",
```

Comment:
You might want to take a look at the table here: https://trino.io/docs/current/security/file-system-access-control.html#catalog-schema-and-table-access 

Basically, if you want people from `aicoe_osc_demo` to run `INSERT INTO` or `DELETE FROM` commands, then they need to have `all` access for catalog.

Reply:
In that case, all groups will need this.